proposals/nnnn-structured-concurrency.md

```diff
@@ -786,6 +793,9 @@ This waiting can be performed either:
 - by the code within the task group itself (e.g., using `next()` repeatedly until it returns `nil`, described below), or
 - implicitly in the task group itself when returning from the `body`.
 
+By default, the task group will schedule child tasks added to the group on the default global concurrent executor. The `runningChildTasksOn` argument can
```

Comment:
I think that's what I expected and sounds good to me 👍 

Doug had some ideas to making it be on the actor's context/exeecutor instead by default... not sure if a good idea tho, as it makes it not parallel then.

Reply:
My understanding was that we expect tasks to be able to change executors as needed for actor-isolated operations. Starting on the global concurrent queue should be the right thing for actor-independent child tasks, but they can change over to actor-specific executors when needed.

Reply:
Hm right, since if a closure needs to touch actor stuff it would hop there... I'm not sure how we can spell "that closure must hop to <actor X>" -- we can spell it as a function on that actor that we call, but I don't think we have a spelling to make a closure itself hop right away? Maybe we don't want to anyway...

```
await group.add { // some executor
  await worker.work() // worker executor
  // hop back // could be optimized away since we only return here
```

for such function would we guarantee that we even submit anything to "some executor"? It may be the one that we'd want to apply the backpressure tricks on... so technically "yes", but in practice... it does not do any work with such simple closure which seems to be likely to be a popular form of it 🤔 What do we want to guarantee there?

So we can have async properties... but they don't become automatically async in an actor, or do they, same like functions?

My example in question was

```
group.add { // some executor
actor.bla // some property; sync
```

I guess if we're making properties also implicitly async then this will become `await actor.bla` I guess and the hops can happen properly... 

I don't know if that is the plan?

Reply:
So I guess:

- no `startingOn` executor set -- we technically could not do that initial hop
- _any_ starting on, always hop initially

Those seem like good rules?